docs/sources/visualizations/text-panel.md

```diff
@@ -15,6 +15,28 @@ weight: 1100
 
 # Text
 
-The text panel visualization lets you make information and description panels for your dashboards.
+The text panel allows you to directly include text/html in your dashboards. This can be used to add contextual information and descriptions, or embed complex HTML.
 
-In **Mode**, select whether you want to use markdown or HTML to style your text, then enter content in the box below. Grafana includes a title and paragraph to help you get started, or you can paste content in from another editor.
+## Mode
+
+The **Mode**, select how the embedded content should be displayed
+
+### Markdown
+
+The content is formatted as [markdown](https://en.wikipedia.org/wiki/Markdown)
+
+### HTML
+
+The content is rendered as [sanitized](https://github.com/grafana/grafana/blob/code-in-text-panel/packages/grafana-data/src/text/sanitize.ts) HTML. If more direct control of the output is needed, the
+[disable_sanitize_html]({{< relref "../setup-grafana/configure-grafana/#disable_sanitize_html" >}}) flag can be set
+and then any html can be entered directly.
+
+### Code
+
+Content is rendered inside a read only code editor. The selected language will provide proper syntax highlighting
```

Comment:
I'm unsure of what you are trying to say in this sentence "The selected language will provide proper syntax highlighting you want to use markdown or HTML to style your text, then enter content in the box below."

Could you rephrase that and I can take another look?

Reply:
🤦 -- that was a garble with the previous language.  Updated and lets see if it makes more sense :)